src/core/parameters/add_channels.js

```diff
@@ -0,0 +1,6 @@
+/* @flow */
+export default {
```

Comment:
that is easy to move, but for the case of subscribe, set_state and unsubscribe are managed from the subscription manager, then can we move these config to subscription manager?